### ISSUE TITLE
Multiple labels exist with the same "for" attribute: #wdn_menu_toggle

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#701]
WCAG2AA
Discovered with: http://squizlabs.github.io/HTML_CodeSniffer/

```
Multiple labels exist with the same "for" attribute. If these labels refer to different form controls, the controls should have unique "id" attributes.
```

Code Snippet:

```
<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
```
